Nexd/Nexd/ui/seeker/RequestConfirmationViewController.swift

```diff
@@ -141,86 +140,67 @@ class RequestConfirmationViewController: ViewController<RequestConfirmationViewC
                 make.height.equalTo(52)
             }
         }
+        let textFields = [firstName, lastName, street, number, zipCode, city, phoneNumber, additionalRequest, deliveryComment]
+        for (index, textView) in textFields.enumerated() {
+            scrollView.addSubview(textView)
+            textView.withBottomBorder()
+            textView.snp.makeConstraints { make in
+                make.height.equalTo(36)
+                make.left.equalToSuperview().inset(13)
+            }
+            textView.tag = index
+            textView.delegate = self
+            if textView == textFields.last {
+                textView.returnKeyType = .done
+            }
+            print("DEBUG: \(textView.tag)")
```

Comment:
- please clean out debug logs and print statements if they are not required for production
- for log statements which should remain even for production please prefer to use `log.debug`, ` log.warning` , `log.error` or `log.verbose`